Makefile

```diff
@@ -28,8 +28,19 @@ install-bin:
 install-npm:
 	npm install && npm fund
 
+TARGET_PATH=target/wasm32-unknown-emscripten/release
+WASM_FILE=$(TARGET_PATH)/resolc.wasm
+JS_FILE=$(TARGET_PATH)/resolc.js
+OUTPUT_FILE=$(TARGET_PATH)/resolc_packed.js
```

Comment:
Nit: Can we rename to something more telling? They are as general as variable names can get..

E.g.

`TARGET_PATH` -> `CARGO_WASM32_EMSCRIPTEN_TARGET_DIR_PATH`
`WASM_FILE` -> `RESOLC_WASM_TARGET_PATH`
`JS_FILE`-> `RESOLC_JS_TARGET_PATH`
`OUTPUT_FILE` -> `RESOLC_JS_PACKED_TARGET_PATH`

Reply:
sure, good idea